mitodl/open-discussions
# Add author_headline to post and comment docs in index

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #1417 

#### What's this PR do?
- Adds `author_headline` to post and comment docs in the search index.
- Updates all relevant post & comment docs when the profile headline is changed.

#### How should this be manually tested?
- Run `docker-compose run web python manage.py recreate_index`, then verify that all the post and comment docs in the index include `author_headline`
- Change the profile headline of a user who has submitted posts and comments.  Then query Elasticsearch to verify that the `author_headline` value has been updated for that user's posts and comments.
